app/controllers/pxe_controller.rb

```diff
@@ -191,7 +191,7 @@ def replace_right_cell(options = {})
     presenter[:right_cell_text] = right_cell_text || @right_cell_text
 
     if !@view || @in_a_form ||
-       (@pages && (@items_per_page == ONE_MILLION || @pages[:items].zero?))
+       (@pages && (@items_per_page == ONE_MILLION || @pages[:items].?zero?))
```

Comment:
`.?` is not valid ruby
`&.` is ;)

Reply:
Let us just pretend this did not happen, OK?